### ISSUE TITLE
Does not work with handlesbar 4.0, only handlesbar 3.0

### ISSUE THREAD

**Issue by the-ica** [XAMPPRocky/fluent-templates#30]
They must have change the trait of their helpers between 4.0 and 3.0. Here is the error : 

```
help: the trait `for<'r, 'reg, 'rc, 's, 't0> Fn<(&'r Helper<'reg, 'rc>, &'reg Handlebars<'reg>, &'rc handlebars::Context, &'s mut RenderContext<'reg, 'rc>, &'t0 mut (dyn handlebars::Output + 't0))>` is not implemented for `FluentLoader<&StaticLoader>`
   = note: required because of the requirements on the impl of `HelperDef` for `FluentLoader<&StaticLoader>`
```

**Comment by XAMPPRocky** [comment]
Thank you for your issue! If someone is interested in tackling this, I'd welcome a PR.

**Comment by XAMPPRocky** [comment]
This is now fixed in 0.7.1
